teachertool/src/components/makecodeFrame.tsx

```diff
@@ -0,0 +1,170 @@
+/// <reference path="../../../built/pxteditor.d.ts" />
+import * as React from "react";
+import { useState, useEffect, useRef } from "react";
+import { ProgressBar } from "react-common/components/controls/ProgressBar";
+
+// This is something I want to do eventually. The iframe should be able to work with a redux store
+// import { connect } from 'react-redux';
```

Comment:
Not a huge concern but I'd recommend removing the commented-out code before checking in. (Could move it to a different branch, if you're worried about losing it).

Reply:
Yeah, good point. I was wanting to leave these in as a reminder, but I have notes and the skillmap example to look back on for extensibility things. I'll take the comments out.